app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(
+            self,
+            name: str,
+            age: int,
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(peoples: list) -> list:
+    for data_person in peoples:
+        Person(
+            name=data_person["name"],
+            age=data_person["age"]
+        )
+
+    for data_person in peoples:
+        person = Person.people[data_person["name"]]
+
+        if "wife" in data_person and data_person["wife"]:
```

Comment:
You could write this check like
```suggestion
        if data_person.get("wife"):
```
It would automatically check that there is a `"wife"` key in the `data_person` dict without raising a `KeyError` and evaluate the boolean type of the value in case it is present.